pyproject.toml

```
@@ -15,15 +15,15 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "vllm"
 authors = [{name = "vLLM Team"}]
-license = { "file"= "LICENSE" }
+license = "Apache-2.0"
+license-files = ["LICEN[CS]E*"]
```

We only have 1 license file

```suggestion
license-files = ["LICENSE"]
```